guides/doc-Release_notes/master.adoc

```diff
@@ -0,0 +1,27 @@
+include::common/attributes.adoc[]
+
+:numbered:
+:toc:
+:toc-placement: left
+
+ifdef::katello[]
+= Foreman {ProjectVersion} and Katello {KatelloVersion} Release Notes
+endif::[]
+
+include::topics/foreman.adoc[leveloffset+=1]
+
+ifdef::katello[]
+include::topics/katello.adoc[leveloffset+=1]
+endif::[]
+
+[appendix]
+= Foreman Contributors
+
+include::topics/contributors-foreman.adoc[leveloffset+=1]
+
+ifdef::katello[]
+[appendix]
+= Katello Contributors
```

Comment:
```suggestion
[id="release_notes_katello_contributers"]
= Katello Contributors
```

Some other headings also omit anchors, maybe add them everywhere.

Reply:
Agreed, that's certainly needed.

Reply:
I added some headers. Please check them.